kea-vscode/src/views/pull-request-contents/commits/commits-root-tree-node.ts

```diff
@@ -39,4 +94,51 @@ export class CommitsRootTreeNode implements IParentTreeNode<CommitTreeNode> {
 
     return commits.map((commit) => new CommitTreeNode(this.#repository, commit));
   };
+
+  #getLocalCommits = async (): Promise<LocalCommitTreeNode[] | null> => {
+    try {
```

Comment:
This logic should be moved to the `KeaRepository` implementation. Update the tests accordingly.